tools/chezmoi/private/BUILD.bazel

```diff
@@ -0,0 +1,73 @@
+# -*- Starlark -*-
+
+"""
+Private BUILD file for the `chezmoi` toolchain.
+
+This file defines the platform-specific toolchain configurations
+and registers them for use in the repository.
+"""
+
+load(":toolchain.bzl", "chezmoi_toolchain")
+
+# Define constraints for different platforms.
+# These are used to select the correct pre-built binary.
+config_setting(
+    name = "darwin_arm64",
+    constraint_values = [
+        "@platforms//os:osx",
+        "@platforms//cpu:arm64",
+    ],
+)
+
+config_setting(
+    name = "darwin_amd64",
+    constraint_values = [
+        "@platforms//os:osx",
+        "@platforms//cpu:x86_64",
+    ],
+)
+
+config_setting(
+    name = "linux_amd64",
+    constraint_values = [
+        "@platforms//os:linux",
+        "@platforms//cpu:x86_64",
+    ],
+)
```

Comment:
should these be defined more centrally available, such is in the `//config` package?